src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -62,7 +68,11 @@ class FilterBam
   @arg(flag='D', doc="If true remove all reads that are marked as duplicates.")   val removeDuplicates: Boolean = true,
   @arg(flag='U', doc="Remove all unmapped reads.")                                val removeUnmappedReads: Boolean = true,
   @arg(flag='M', doc="Remove all mapped reads with MAPQ lower than this number.") val minMapQ: Int = 1,
-  @arg(flag='S', doc="Remove all reads marked as secondary alignments.")          val removeSecondaryAlignments: Boolean = true
+  @arg(flag='P', doc="Remove all reads that are not part of a mapped pair.")      val removeUnpairedReads: Boolean = false,
+  @arg(flag='S', doc="Remove all reads marked as secondary alignments.")          val removeSecondaryAlignments: Boolean = true,
+  @arg(          doc="Remove all reads with insert size < this value.")           val minInsertSize: Option[Int] = None,
+  @arg(          doc="Remove all reads with insert size > this value.")           val maxInsertSize: Option[Int] = None,
+  @arg(flag='m', doc="Remove reads with fewer than this many mapped bases.")      val minMappedBases: Option[Int] = None
```

Comment:
Are you concerned with the documentation or the actual handling of the options?  In my mind the options are inclusive already.  I.e. if you specify `--min-insert-size=100 --max-insert-size=600` then you will retain all reads with inserts in (100,600) inclusive.  Similarly for `minMappedBases` if you specify `--min-mapped-bases=50` then you'll retain records with >= 50 mapped bases.

It's just that I think the options are documented more accurately as removing things below the minimum or above the maximum, because e.g. if `--min-insert-size` was documented as "retain records with insert size >= this value", it would be confusing when combined with `--max-insert-size`.